impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.nodelegateinjectionpoint;
+package org.jboss.cdi.tck.tests.full.decorators.broken.nodelegateinjectionpoint;
```

Comment:
Should be `full.decorators.custom.broken.nodelegateinjectionpoint`, not `full.decorators.broken.nodelegateinjectionpoint`.

Reply:
The other issues are fixed, but this one remains.

Reply:
This wasn't addressed if I understand correctly.

Reply:
Ok, #280 add this additional rename